tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -156,19 +156,55 @@ spec:
                   ORAS_ARGS+=("--platform $os/$arch")
                 fi
 
+                # Save the OCI manifest locally, to pass to a script to create the pyxis entry
+                MANIFEST_FILE="/tmp/oras-manifest-fetch.json"
                 oras manifest fetch \
                   --registry-config "$AUTH_FILE" \
                   "${ORAS_ARGS[@]}" \
                   "${PULLSPEC}" \
-                      | tee /tmp/oras-manifest-fetch.json
+                      | tee "${MANIFEST_FILE}"
+
+                # Augment that manifest with further information about the layers, decompressed
```

Comment:
I think it would be nice to cover this part in the task tests. Also, the mock manifest fetch should return more than just what we have now: https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/create-pyxis-image/tests/mocks.sh#L63

It doesn't even have a mediaType for the layer it returns. So then the "if blob_type =~ zip" is run against an empty string.

Reply:
I agree, thanks! I'll work on that.

Reply:
How is d70fbcfa18590cbd27d17ee3068c1c26cd7724d4?

Reply:
It looks good, thanks!